templates/base/head_script.tmpl

```diff
@@ -8,6 +8,7 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 	window.config = {
 		appUrl: '{{AppUrl}}',
 		appSubUrl: '{{AppSubUrl}}',
+		getCurAbsUrl: (relUrl) => `${window.location.origin}{{AppSubUrl}}${relUrl}`,
```

Comment:
This is now unused, remove?

Reply:
Now it has been used when copying a comment link.